src/include/rados/librados.hpp

```diff
@@ -1153,6 +1153,7 @@ namespace librados
     int get_object_pg_hash_position2(const std::string& oid, uint32_t *pg_hash_position);
 
     config_t cct();
+    IoCtxImpl *get_impl() {return io_ctx_impl;}
```

Comment:
I don't think it's a good idea to expose implementation to user. Maybe adding an interface for what you want is better.